src/App.css

```diff
@@ -48,14 +48,158 @@ input {
   box-shadow: 1px 1px 2px black;
 }
 
-button {
+.homeWrap {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  background-color: lightblue;
+  height: 50vh;
+  width: 30vw;
+  justify-content: space-around;
+
+}
+
+.joinLink {
+  text-decoration: none;
+  color: black;
+  margin: 3vh;
+  padding: 1vh;
+  border: 2px solid black;
+  border-radius: 2px;
+  background-color: white;
+  font-size: 25px;
+}
+
+.joinLink:hover {
+  font-size: 26px;
+  box-shadow: 1px 1px 2px black;
+
+}
+
+.newListLink {
+  color: black;
+  text-decoration: none;
+  border: 2px solid black;
+  margin: 1vh;
+  border-radius: 2px;
+  background-color: white;
+  padding: 1vh;
+
+}
+
+.newListLink:hover {
+  box-shadow: 1px 1px 2px black;
+}
+
+.joinListButton {
+  color: black;
+}
+
+.tokenButton {
+  position: relative;
   background-color: white;
   border: 2px solid black;
   padding: 1vh;
   margin: 1vh;
+  height: 10vh;
+  border-radius: 2px;
 }
 
-button:hover {
-  font-size: 23px;
+.tokenButton:hover {
+  font-size: 32px;
   box-shadow: 1px 1px 2px black;
+  transition: .1s;
+  transition-timing-function: ease-in;
+}
```

Comment:
Not that it affects functionality - but button:hover makes the button smaller which seems counter-intuitive. But maybe you two meant to do that though...?

Reply:
@ClaudiaStar do you mean the deleted button from lines 58-59, or one of the buttons that the app is still using?

Reply:
Sorry I commented on the wrong part of the code. I mean the Get Token button. The styling looked a little bit funky on hover, but then again maybe that was intentional?

Reply:
Looking at the .tokenButton and .tokenButton:hover lines, on 98 and 108, I don't see anything funky. Can you be more specific?

Reply:
I just pulled your branch again and it looks different than last time I looked at it. Everything looks great, so don't mind my comment!